app/controllers/v0/pension_claims_controller.rb

```diff
@@ -42,7 +42,7 @@ def create
       pension_monitor.track_create_attempt(claim, current_user)
 
       in_progress_form = current_user ? InProgressForm.form_for_user(claim.form_id, current_user) : nil
-      claim.itf_datetime = in_progress_form.created_at if in_progress_form
+      claim.form_start_date = in_progress_form.created_at if in_progress_form
```

Comment:
does the schema need to be updated?

Reply:
it already was in a previous PR: https://github.com/department-of-veterans-affairs/vets-api/pull/17044/files. I'm gonna need to add a follow up PR to move the data from itf_datetime to form_start_date, then remove the itf_datetime field.